.travis.yml

```diff
@@ -1,28 +1,13 @@
-sudo: false
 language: ruby
 rvm:
-  - 1.9
-  - 2.0
-  - 2.1
-  - 2.2
-  - 2.3.1
+  - 2.2.7
```

Comment:
Can we drop 2.2 as well?

Reply:
why? Rails 5.0+ and 5.2+ is still 2.2.2+ We're targeting that: https://rubygems.org/gems/rails/

Reply:
Ruby 2.2 will [officially be EOL at the end of March 2018](https://www.ruby-lang.org/en/news/2017/04/01/support-of-ruby-2-1-has-ended/), and so I saw this as an opportunity to prune. I would like to be in the habit of aggressively dropping support for EOL Rubies. Even if Rails supports it, I'd be fine dropping it now, a couple months ahead of schedule.

Reply:
If Rails still supports 2.2.2 I don't see why we should restrict it. Maybe some still choose to run 2.2 for some crazy reason. When Rails drops it, we can drop it too.

Reply:
I'm for supporting what Rails supports, back to Rails 5.0 as we've agreed.